.travis.yml

```diff
@@ -28,39 +28,47 @@ matrix:
         - qmake
         - make install INSTALL_ROOT=$WORKDIR/build
         - androiddeployqt --output $WORKDIR/build
+  - os: osx
+    script: 
+      - QT_CI_PACKAGES=qt.qt5.597.clang_64 PATH=${TRAVIS_BUILD_DIR}/recipes:${TRAVIS_BUILD_DIR}/bin:${PATH} install-qt-osx 5.9.7 && source qt-5.9.7.env && qmake -version
```

Comment:
maybe use only one script for testing on mac and linux platforms?

Reply:
I agree they could be combined but I haven't chased that.  I also don't have a mac so all of this is only being run on travis which makes development slow.

Can we agree on the spelling of QT_TARGET_CATALOG vs. QT_target_catalog? Or should it really be QT_CI_TARGET_CATALOG?